app/controllers/application_controller.rb

```diff
@@ -50,4 +50,11 @@ def configure_permitted_parameters
   def after_sign_in_path_for(_resource)
     sections_path
   end
+
+  def permitted_user_sign_up_parameters
```

Comment:
Maybe you can call it `permitted_sign_up_parameters` since all sign-ups are for users :)